data/cisagov_R.yml

```diff
@@ -34,4 +34,596 @@ software:
     references:
       - ''
     last_updated: '2022-01-12T07:18:55+00:00'
+  - vendor: Red Hat
+    product: Red Hat Enterprise Linux 6 Extended Lifecycle Support
+    cves:
+      cve-2021-4104:
+        investigated: true
```

Comment:
You can replace lines 41-46 with above.  It looks like you just repeated the Product name was the affected/fixed/or unaffected versions.  If there is a specific version, or if you wanted to put 'All' or something like that, that is what is typically done.  Also the links go under vendor_links like you have put on lines 47-48.  Remove link of line 45.